contracts/TreasuryActions.cdc

```diff
@@ -60,6 +60,10 @@ pub contract TreasuryActions {
     pub let amount: UFix64
 
     pub fun execute(_ params: {String: AnyStruct}) {
+      pre {
+        self.amount > 0.0 : "Amount should be higher than 0.0"  
```

Comment:
i think it might be more appropriate to put this `pre` condition inside the `init` function for these actions.